addons/ai_hearing/XEH_PreInit.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+#include "XEH_PREP.sqf"
+
+[
+    "TFAR_AICanHearPlayer",
+    "CHECKBOX",
+    [LSTRING(SETTING_HEADER), LSTRING(SETTING_DESC)],
+    "Task Force Arrowhead Radio",
```

Comment:
Don't we have a stringtable entry for that?

Reply:
This is the not localized Category Name.

Reply:
Welll.. On the other end it's a Name.
Names usually don't get translated